src/components/CourseForm/CourseForm.jsx

```diff
@@ -11,10 +11,12 @@
 // //   ** user should be able to remove author from the course
 // //   ** add validation to the fields
 // //   ** add new course to the 'coursesList' and navigate to the '/courses' page => new course should be in the courses list
+// ** TASK DESCRIPTION ** - https://d17btkcdsmqrmh.cloudfront.net/new-react-fundamentals/docs/module-2/home-task/components#add-new-course
```

Comment:
I think here and further should be a double comment.

Reply:
fixed all cases